README.md

```diff
@@ -82,7 +82,7 @@ const auth0 = new Auth0Client({
 
 //if you do this, you'll need to check the session yourself
 try {
-  await getTokenSilently();
+  await checkSession();
 } catch (error) {
   if (error.error !== 'login_required') {
```

Comment:
Isnt this a bit weird now? `checkSession` will never throw `login_required` (https://github.com/auth0/auth0-spa-js/blob/master/src/Auth0Client.ts#L575) as it is part of the `RECOVERABLE_ERRORS` https://github.com/auth0/auth0-spa-js/blob/master/src/constants.ts. So it will never throw login_required.

Reply:
Very true, let me revert